dradis-nexpose.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'dradis-plugins', '~> 3.6'
   spec.add_dependency 'nokogiri', '~> 1.3'
 
-  spec.add_development_dependency 'bundler', '~> 1.6'
+  spec.add_development_dependency 'bundler', '>= 1.6'
```

Comment:
@dormi why is this needed?

Reply:
when running specs, if bundler 1.6 is not installed it complains. we don't need that constraint.